packages/react-components/src/components/RichTextEditor/Plugins/CopyPastePlugin.tsx

```diff
@@ -50,6 +53,46 @@ const handleBeforePasteEvent = (
   }
 };
 
+/* @conditional-compile-remove(rich-text-editor-image-upload) */
+/**
+ * @internal
+ * Exported only for unit testing
+ */
+export const handleInlineImage = (
+  event: PluginEvent,
+  onUploadImage?: (image: string, fileName: string) => void
+): void => {
+  // We don't support the pasting options such as paste as image yet.
```

Comment:
For this comment, do you mean pasting files as images?

Reply:
> For this comment, do you mean pasting files as images?

In RoosterJS, there's a context menu that let users to choose the pasted format such as pasteAsImage. We don't support those options yet. I can remove this comment since it's causing more confusion.